packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/alb/application-load-balancer.ts

```diff
@@ -265,15 +265,33 @@ export class ApplicationLoadBalancer extends BaseLoadBalancer implements IApplic
     * See https://docs.aws.amazon.com/elasticloadbalancing/latest/application/enable-access-logging.html#bucket-permissions-troubleshooting
     */
 
+    if (bucket.encryptionKey) {
+      throw new Error('Encryption key detected. Bucket encryption using KMS keys is unsupported');
+    }
+    super.logAccessLogs(bucket, prefix);
```

Comment:
This PR adds a method with the same content as `BaseLoadBalancer.logAccessLogs` to `ApplicationLoadBalancer` to perform validation for ALB only.
However, modifying both methods would be necessary when changing the access log settings, so I changed it to call the base class implementation except for the validation.
https://github.com/aws/aws-cdk/pull/29382

Reply:
Isn't this change fundamentally not related to this PR for the connection logs? If so, it would be better to use a separated PR.

If not, please let me know.